scripts/linux/pkg/deb/assets/DEBIAN/postinst

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# postinst script for ravendb
+#
+# see: dh_installdeb(1)
+
+set -e
+
+if [ ! -z "$_DEB_DEBUG" ]; then
+    set -x
+fi
+
+# summary of how this script can be called:
+#        * <postinst> `configure' <most-recently-configured-version>
+#        * <old-postinst> `abort-upgrade' <new version>
+#        * <conflictor's-postinst> `abort-remove' `in-favour' <package>
+#          <new-version>
+#        * <postinst> `abort-remove'
+#        * <deconfigured's-postinst> `abort-deconfigure' `in-favour'
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see https://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+case "$1" in
+    configure)
+    ;;
+
+    abort-upgrade|abort-remove|abort-deconfigure)
+    ;;
+
+    *)
+        echo "postinst called with unknown argument \`$1'" >&2
+        exit 1
+    ;;
+esac
+
+(id -u ravendb >/dev/null 2>&1 \
+    || adduser --system --shell /bin/false --disabled-password --disabled-login --group ravendb) 
+
+chown root:ravendb /usr/bin/ravendb 
+chmod 555 /usr/bin/ravendb 
+
+ln -sf /usr/local/ravendb/server/rvn /usr/bin/rvn
+
+mkdir -p /var/log/ravendb/{logs,audit} /var/lib/ravendb/{data,security,nuget} 
+chown root:ravendb /var/log/ravendb/{logs,audit} /var/lib/ravendb/{data,security,nuget} 
+chmod 770 /var/log/ravendb/{logs,audit} /var/lib/ravendb/{data,security,nuget} 
+
+find /usr/local/ravendb -type d -exec chmod 555 {} + 
+find /usr/local/ravendb -type f -exec chmod 444 {} + 
+
+chown root:ravendb /usr/local/ravendb/server/{Raven.Server,rvn}
+chmod 550 /usr/local/ravendb/server/{Raven.Server,rvn}
+
+if command -v setcap &> /dev/null; then
+    setcap CAP_NET_BIND_SERVICE=+eip /usr/local/ravendb/server/Raven.Server
+fi
+
+# Do not overwrite existing settings file if it's there
+if [ ! -f /etc/ravendb/settings.json ]; then
+    cp /usr/local/ravendb/settings.template.json /etc/ravendb/settings.json
+    chown root:ravendb /etc/ravendb/settings.json 
+    chmod 660 /etc/ravendb/settings.json 
+fi
+
+if command -v systemctl &> /dev/null; then
+    cp /usr/local/ravendb/ravendb.service /etc/systemd/system/ravendb.service 
+    chmod 444 /etc/systemd/system/ravendb.service
+
+    systemctl daemon-reload 
+    systemctl reenable ravendb
+    systemctl restart ravendb
+fi
+
+# CopyPalFile() workaround - 4.2 only
```

Comment:
@ppekrol Do you recall any other files being copied in this manner in runtime? We need to handle this ahead of time on installation when we're `root`.

Reply:
I think we have zstd and sodium as well